### DISCUSSION TITLE
Change file format?

### DISCUSSION THREAD

**Discussion by panthro100** [spatie/laravel-medialibrary#3293] [Q&A]
I need to create `webp` images, I have tried the following without any success:

```
 $model->addMediaFromUrl($url)->format(Manipulations::FORMAT_WEBP)->toMediaCollection();
```

Am I correct thinking that it needs to be added as a `conversion`?

```
$this->addMediaConversion('thumb')->format(Manipulations::FORMAT_WEBP)->withResponsiveImages();
```

When I do this, I store the original uploaded file (`jpeg`) and an extra converted file under `conversions`, which is the original `jpeg` converted to `webp`, and then the various responsive image versions of the `webp` in `webp` format in the `responsive image` folder.

Is there a way to just store the responsive `webp` images? Without storing the original converted `webp` file in the `conversions` folder?

**Comment by hesammoosapour** [comment]
I do this : 
` $this
            ->addMediaConversion('poster')
            ->performOnCollections('thumb')
            ->withResponsiveImages()
            ->nonQueued()
            ->format('WEBP');`
            
Can you delete media afterwards?
